### DISCUSSION TITLE
Gholami et al. 2023 "Federated learning for diagnosis of age-related macular degeneration" in Frontiers in Medicine

### DISCUSSION THREAD

**Discussion by mxochicale** [oocular/ready#106] [References]
> Gholami, Sina, Jennifer I. Lim, Theodore Leng, Sally Shin Yee Ong, Atalie Carina Thompson, and Minhaj Nur Alam. "Federated learning for diagnosis of age-related macular degeneration." Frontiers in Medicine 10 (2023): 1259017.
* Abstract: This paper presents a federated learning (FL) approach to train deep learning models for classifying age-related macular degeneration (AMD) using optical coherence tomography image data. We employ the use of residual network and vision transformer encoders for the normal vs. AMD binary classification, integrating four unique domain adaptation techniques to address domain shift issues caused by heterogeneous data distribution in different institutions. Experimental results indicate that FL strategies can achieve competitive performance similar to centralized models even though each local model has access to a portion of the training data. Notably, the Adaptive Personalization FL strategy stood out in our FL evaluations, consistently delivering high performance across all tests due to its additional local model. Furthermore, the study provides valuable insights into the efficacy of simpler architectures in image classification tasks, particularly in scenarios where data privacy and decentralization are critical using both encoders. It suggests future exploration into deeper models and other FL strategies for a more nuanced understanding of these models' performance. Data and code are available at https://github.com/QIAIUNCC/FL_UNCC_QIAI
* Github https://github.com/QIAIUNCC/FL_UNCC_QIAI
* Goggle-citations: https://scholar.google.com/scholar?cites=1252819767913554138&as_sdt=2005&sciodt=0,5&hl=en

### Relevant figures

> A notable observation was the inconsistent performance of the local DS1 ResNet18 model across different test sets. While this model demonstrated commendable efficacy on its native and DS2 test sets, it faltered with DS3. **This challenge arose from the brightness distribution disparity among DS1, DS2, and DS3 (Fig3)**

<img width="932" height="384" alt="image" src="https://github.com/user-attachments/assets/42ee9fbd-6004-4ba7-8744-7403c668b904" />

> In the FL context, the performance of FedAvg, FedProx, and
FedSR models, all utilizing a ResNet18 encoder, was found lacking
on DS3’s test set. This was unexpected, especially since FedProx and
FedSR were crafted to counter domain shifts. **This performance
gap is rooted in data heterogeneity, which induces a drift in the
learning trajectory.** This drift, primarily aligned with DS1 and DS2,
results in suboptimal outcomes when the aggregated FL model is
tested on DS3. Interestingly, despite its modest performance on
DS3, the FedSR ResNet18 model excelled across all E-values on
DS2’s test set.

<img width="932" height="384" alt="image" src="https://github.com/user-attachments/assets/46e9ac17-1719-49bb-813f-be7bdc94c674" />
